sdk/src/main/java/com/mapbox/maps/FontUtils.kt

```diff
@@ -39,7 +39,7 @@ internal object FontUtils {
       }
     }
     Log.i(
-      TAG, "Couldn't map font family for local ideograph, using ${DEFAULT_FONT} instead"
+      TAG, "Couldn't map font family for local ideograph, using $DEFAULT_FONT instead"
```

Comment:
strange that it was not caught by the CI?

Reply:
CI was not run for https://github.com/mapbox/mapbox-maps-android/pull/534 as it was opened not in our repo but in forked repo.
Will be cherry-picking such PRs in future in order to avoid such situations.